sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -2855,9 +2859,14 @@ void CNPC_Combine::BuildScheduleTestBits( void )
 #endif
 
 #ifdef EZ2
-	if ( !IsCurSchedule( SCHED_COMBINE_ORDER_SURRENDER ) && !IsCurSchedule( SCHED_COMBINE_PRESS_ATTACK ) && !m_StandoffBehavior.IsRunning() /*&& !IsCurSchedule( SCHED_COMBINE_ESTABLISH_LINE_OF_FIRE ) && !IsCurSchedule( SCHED_MOVE_TO_WEAPON_RANGE )*/ )
+	if ( !m_StandoffBehavior.IsRunning() )
 	{
-		SetCustomInterruptCondition( COND_COMBINE_CAN_ORDER_SURRENDER );
+		if ( !IsCurSchedule( SCHED_COMBINE_ORDER_SURRENDER ) && !IsCurSchedule( SCHED_COMBINE_PRESS_ATTACK ) /*&& !IsCurSchedule( SCHED_COMBINE_ESTABLISH_LINE_OF_FIRE ) && !IsCurSchedule( SCHED_MOVE_TO_WEAPON_RANGE )*/ )
+		{
+			SetCustomInterruptCondition( COND_COMBINE_CAN_ORDER_SURRENDER );
```

Comment:
Why is this custom interrupt condition only set now if the schedule isn't already running? That seems logical but I'm not sure why it's included with the obstruction behavior

Reply:
Nothing actually changed with the way `COND_COMBINE_CAN_ORDER_SURRENDER` was set, the if statement for checking if standoff behavior is active was moved because obstruction behavior shouldn't interrupt it either.

Reply:
I see the removal from the above line now - I misread it, my bad